README.md

```diff
@@ -47,10 +47,10 @@ export default Ember.Controller.extend({
 
 The `cookies` service has methods for reading and writing cookies:
 
-* `read(name)`: reads the cookie with the given name, returns its value as a
-  `String`.
+* `read(name, options = {})`: reads the cookie with the given name, returns its value as a
+  `String`; options can be used to set `useRawValue` (boolean).
```

Comment:
we should add something like _"…set `raw` which disables (boolean) URL-decoding"_